lib/tasks/health.rake

```diff
@@ -210,108 +209,27 @@ namespace :health do
 
   # DB related, provides health:db:migrate etc.
   namespace :db do |ns|
-
-    task :drop do
-      Rake::Task["db:drop"].invoke
-    end
-
-    task :create do
-      Rake::Task["db:create"].invoke
-    end
-
-    task :setup do
-      Rake::Task["db:setup"].invoke
-    end
-
-    task :migrate do
-      Rake::Task["db:migrate"].invoke
-    end
-
-    namespace :migrate do
-      task :redo do
-        Rake::Task["db:migrate:redo"].invoke
-      end
-      task :up do
-        Rake::Task["db:migrate:up"].invoke
-      end
-      task :down do
-        Rake::Task["db:migrate:down"].invoke
-      end
-      task :status do
-        Rake::Task["db:migrate:status"].invoke
-      end
-    end
-
-    task :rollback do
-      Rake::Task["db:rollback"].invoke
-    end
-
-    task :seed do
-      Rake::Task["db:seed"].invoke
-    end
-
-    task :version do
-      Rake::Task["db:version"].invoke
-    end
-
     namespace :schema do
-      task :load do
-        Rake::Task["db:schema:load"].invoke
-      end
-
-      task :dump do
-        Rake::Task["db:schema:dump"].invoke
-      end
-
       desc "Conditionally load the database schema"
       task :conditional_load, [] => [:environment] do |t, args|
-        if HealthBase.connection.table_exists?(:schema_migrations)
-          puts "Refusing to load the health database schema since there are tables present. This is not an error."
+        if HealthBase.connection.tables.length == 0
+          Rake::Task['db:schema:load:health'].invoke
         else
-          Rake::Task['health:db:schema:load'].invoke
+          puts "Refusing to load the health database schema since there are tables present. This is not an error."
         end
       end
     end
 
     namespace :structure do
-      task :load do
-        Rake::Task["db:structure:load"].invoke
-      end
-
-      task :dump do
-        Rake::Task["db:structure:dump"].invoke
-      end
-
       desc "Conditionally load the database structure"
       task :conditional_load, [] => [:environment] do |t, args|
-        if HealthBase.connection.table_exists?(:schema_migrations)
-          puts "Refusing to load the health database structure since there are tables present. This is not an error."
+        if HealthBase.connection.tables.length == 0
```

Comment:
This may have changed with rails 6 but I thought we went to `table_exists?(:schema_migrations)` because sometimes there are tables that get created when the db is created (maybe postgres or stats or something.)  Here's the prior commit:
https://github.com/greenriver/hmis-warehouse/blame/6572fabcd371043b80879227b446de4ec96b5af7/lib/tasks/health.rake#L268
I'm guessing this was just a lack of updating the old rails 6 branch.

Reply:
I'll check on this.